src/documentation/search/SearchDialog.tsx

```diff
@@ -42,6 +46,47 @@ const SearchDialog = ({
     }
   }, [onClear]);
 
+  useEffect(() => {
+    const keydown = (e: KeyboardEvent) => {
+      if (e.key === "ArrowUp") {
+        e.preventDefault();
+        const activeEl = document.activeElement;
+        const siblingEl =
+          activeEl?.parentElement?.previousElementSibling?.firstElementChild;
+        if (siblingEl) {
+          (siblingEl as HTMLElement).focus();
+        } else {
+          const siblingInPrevSection =
+            activeEl?.parentElement?.parentElement?.previousElementSibling
+              ?.lastElementChild?.firstElementChild;
+          if (siblingInPrevSection) {
+            (siblingInPrevSection as HTMLElement).focus();
+          }
+        }
+      } else if (e.key === "ArrowDown") {
+        e.preventDefault();
+        const activeEl = document.activeElement;
+        const siblingEl =
+          activeEl?.parentElement?.nextElementSibling?.firstElementChild;
+        if (siblingEl) {
+          (siblingEl as HTMLElement).focus();
+        } else {
+          const siblingInNextSection =
+            activeEl?.parentElement?.parentElement?.nextElementSibling
+              ?.firstElementChild?.nextElementSibling?.firstElementChild;
+          if (siblingInNextSection) {
+            (siblingInNextSection as HTMLElement).focus();
+          }
+        }
+      }
+    };
+
+    document.addEventListener("keydown", keydown);
+    return () => {
+      document.removeEventListener("keydown", keydown);
+    };
+  }, []);
+
```

Comment:
@microbit-matt-hillsdon This is horrible. This overrides the default behaviour of being able to scroll through the search results using the up and down arrows, which on reflection, is not something I'm sure we want to do? If we do want this, perhaps there is a better way?

Reply:
Yeah, worth some further thought. Everything else here is more straightforwardly valuable (and fixed tabbing) so perhaps split this bit out and let's get the rest in?